stories/25-Modal.stories.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react'
+import { MLModal, MLButton } from 'marklogic-ui-library'
+import { withKnobs, text, boolean } from '@storybook/addon-knobs'
+import {
+  DraggableModalProvider,
+} from 'ant-design-draggable-modal'
+import { withActions } from '@storybook/addon-actions'
+
+export default {
+  title: 'Feedback/MLModal',
+  component: MLModal,
+  decorators: [withKnobs, withActions('click')],
```

Comment:
@phoenixeliot This works for clicks on the buttons I've made to launch the modals, but not on the modal buttons themselves. Any insight into why that might be?

Reply:
I suggest binding the actions directly like so: https://github.com/phoenixeliot/design-system/blob/aaac6c1d1dab3da41f2ae32f2c071e35fa6245bd/stories/2-Button.stories.js#L16

Reply:
Could do, but there will end up being a lot of places that needs to happen, and since they also have handlers already (unlike the Button stories), they will have to be a little more complicated.

Reply:
yeah, I guess I suggest having eg `onOk` take something like `{(e) => { hide(); return action(e) }}`

Reply:
Writing the handler inline that way didn't work but I put the action in the `hide` callback (and friends) and it's working. I'm not sure how instructive it is to have it, but it's there.